test/cts/tool/CTSConverter/src/main.py

```diff
@@ -97,55 +97,79 @@ def create(opath, file_dict, file_list, describe):
   output_path_root = "./output"
   if not os.path.exists(output_path_root):
     os.makedirs(output_path_root)
 
-  output_file_all = os.path.join(output_path_root, args_all)
+  (output_file_all_path, output_file_all_name) = os.path.split(args_all)
+  if output_file_all_path == "":
+    output_file_all = os.path.join(path_root, args_all)
+  else :
+    output_file_all = args_all
 
   describeString = "CTS"
 
   file_dict_all = dict()
 
   if not args_transfer == "-":
-    output_path_transfer = os.path.join(output_path_root, "cts")
-    if os.path.exists(output_path_transfer):
-      del_file(output_path_transfer, False)
+    output_path_transfer_clear = os.path.join(output_path_root, "cts")
+    if os.path.exists(output_path_transfer_clear):
+      del_file(output_path_transfer_clear, False)
     else:
-      os.makedirs(output_path_transfer)
+      os.makedirs(output_path_transfer_clear)
 
-    support_cts_file = "./slice.txt"
+    output_path_transfer = os.path.join(output_path_transfer_clear, "specs")
+    support_cts_file = "./slice.json"
 
     with open(support_cts_file) as cts_file:
-      cts_file_names = cts_file.readlines()
+      cts_file_names = json.load(cts_file)
       print ("Open support cts file: " + support_cts_file + "\n")
 
     print ("transfer nn test case to js test case....\n")
     transfer(args_transfer, output_path_transfer, cts_file_names)
 
   if not args_cts == "-":
-    print ("scan test cts directory....\n")
+    print ("scan test cts directory....")
     cts_file_dict = get_file_names(args_cts, ".js")
     file_dict_all.update(cts_file_dict)
 
   if not args_supplement == "-":
-    print ("scan test supplement directory....\n")
+    print ("scan test supplement directory....")
 
     describeString = "CTS Supplement Test"
 
     supplement_file_dict = get_file_names(args_supplement, ".js")
     file_dict_all.update(supplement_file_dict)
 
   if not args_plus == "-":
-    print ("scan test plus directory....\n")
+    print ("scan test plus directory....")
     plus_file_dict = get_file_names(args_plus, ".js")
     file_dict_all.update(plus_file_dict)
 
   if not args_all == "-":
-    print ("reordering by name....\n")
+    skip_files = ["cts-all.js", "cts_supplement-all.js", "conv_65_65_96.js"]
+    (args_all_path, args_all_name) = os.path.split(args_all)
+    skip_files.append(args_all_name)
+
+    for name in skip_files:
+      if name in file_dict_all:
+        print ("skip creating file name: %s"%file_dict_all[name])
+        del file_dict_all[name]
+
+    print ("reordering by name....")
     file_list = sorted(file_dict_all.keys())
 
-    print ("create all test case file....\n")
+    print ("create all test case file: %s"%str(output_file_all))
     create(output_file_all, file_dict_all, file_list, describeString)
 
-  print ("transfer and create all files are completed")
+  if not args_transfer == "-":
+    if not args_all == "-":
+      print ("transfer and create all files are completed")
```

Comment:
Please modify **transfer and create all files are completed** to **Completed to transfer Android CTS Spec files.**